project_task_note/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Tecnativa Carolina Fernandez
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Add notes in Project Task",
```

Comment:
This text is shorter from the summary (or just put the name, not both).

```suggestion
    "name": "Project task notes",
```